tests/integration/test_integration.py

```diff
@@ -28,7 +28,6 @@ async def _deploy_grafana_agent(ops_test: OpsTest):
 
 
 @pytest.fixture(scope="module")
-@pytest.mark.abort_on_fail
```

Comment:
Why do want to remove this?

Reply:
It does nothing, according to the [documentation](https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function). We already have the marker on the actual test.